examples/test_parallel/parallel_simple2.zls

```diff
@@ -0,0 +1,4 @@
+let node exec () = 
```

Comment:
Add brief description of what is being tested, it would be good to have the generated vc as a comment.